### PR TITLE
sunpy 4.1

### PR DESCRIPTION
**Pull request by dstansby**
**I would like two independent reviews on this, specifically to check that the dependencies I've updated are okay.** They have been updated to match the dependencies in `setup.cfg`, and re-organised so it's easier to compare this file and `setup.cfg`.

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by dstansby** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -31,39 +31,46 @@ requirements:
   run:
     - python
     - setuptools
```
Comment (Cadair) [inline recipe/meta.yaml:33]: Is setuptools a runtime dependency still? I think we have solved the version issue I think we added this for?

**Review by Cadair — APPROVED** [review]

**Comment by nabobalis** [comment]
Side note, do we need to clean up the older 3 branches and create a 4.0 one?

**Comment by dstansby** [comment]
Good shout, not sure why we should remove older branches (given there's no tags pointing to them I think it's best to keep them around just incase) but I will create a new 4.0.x branch

**Review by nabobalis — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (dstansby) [reply recipe/meta.yaml]: I don't think so looking at `setup.cfg`? I've just removed it from here, lets see if it builds fine.

Comment (dstansby) [reply recipe/meta.yaml]: Looks 👍 without

**Comment by dstansby** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 2]
```diff
@@ -30,40 +30,49 @@ requirements:
...
+    - opencv
     - pytest-astropy
     - pytest-mock
-    - pytest-xdist
-    - asdf !=2.10.0
-    - jplephem
     - pytest-mpl
-    - reproject
-    - cdflib
+    - pytest-xdist
```
Comment (Cadair) [inline recipe/meta.yaml:74]: Why do we need this? (not that it's an issue, I am just curious)

**Review by Cadair — APPROVED** [review]

**Pull request merged**